File: C#/백준/Bronze/23235. The Fastest Sorting Algorithm In The World/The Fastest Sorting Algorithm In The World.cs
using System;

namespace q{
    class w{
        static void Main(){
            int i=0;
            while(true){
                string[] aa = Console.ReadLine().Split();
                int a = int.Parse(aa[0]);
                if(a==0){
                    break;
                }
                else{
                    Console.WriteLine($"Case {i+1}: Sorting... done!");
                }
                i+=1;
            }
        }
    }
}

File: Java/백준/Bronze/23235. The Fastest Sorting Algorithm In The World/The Fastest Sorting Algorithm In The World.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in); int i=0;
        while(true){
            String a = in.nextLine();
            if(a.equals("0")){
                break;
            }
            else{
                System.out.printf("Case %d: Sorting... done!%n",i+1);
            }
            i+=1;
        }
    }
}